### ISSUE TITLE
How NOT to use builds/* system

### ISSUE THREAD

**Issue by abulte** [kraftwagen/kraftwagen#29]
Hi,

I'm looking for a way to NOT keep an archive of all the builds and just build and keep the last version (for various reasons).

It seems to be (have been) possible (see comment below), but I can't find the way to do so either with testing configurations or by looking at the code of `build.kw.inc`

```
  // 2. 'build' directory. If we have name for a directory to store a single
  // build, use this. This is the default behavior and will succeed in most
  // case. If you want to disable this logic, for example in a production
  // environment, you should set 'build-dir' to FALSE in kraftwagenrc.php.
```

**Comment by jerbob92** [comment]
It will also break on Vagrant because of the symlinks. You won't be able to edit the files locally as the symlink target doesn't exist locally when the directory is different inside your vm.
There are 2 solutions for this:
  * Don't use symlinks at all
  * Use relative symlinks

**Comment by bighappyface** [comment]
@jerbob92 I use Vagrant with KW and, while the symlink only works within the VM where I perform the builds, I am able to edit the files locally by means of the mounted directory and the "builds" directory.

Here are the steps:

1. Clone your KW project into a subdir "src" `git clone <repo> project/src`
2. Change directories into "project/src"
3. Setup the project with `drush kw-s`
4. Navigate into the outer "project" directory
5. Booth VM with current directory mounted to "/vagrant" (or whatever you prefer)

Again, the build symlink won't work correctly being an absolute symlink, but you should be able to share the files between the VM and host without any difficulty.

See https://github.com/kraftwagen/kraftwagen/issues/21 for more background on relative symlinks for KW

**Comment by jerbob92** [comment]
@bighappyface You are right, it does work like that, we already do that as a workaround.
Thing is I expect it to work like locally. Changing your IDE directory every time you do a new build isn't really nice and it might be something you forget. 

There is also another workaround: using the same path within the VM as outside, that makes the symlinks work. 

Too bad the relative symlinks issue wasn't really "fixed".

**Comment by jerbob92** [comment]
Added relative symlinks in #40

**Comment by bighappyface** [comment]
https://github.com/kraftwagen/kraftwagen/pull/40 is closed and symlinks are in!

**Comment by handrus** [comment]
I will try to take this over this weekend, not a hard promise but stay tuned ;)
